README.md

```diff
@@ -19,6 +19,16 @@ MONAI is a [PyTorch](https://pytorch.org/)-based, [open-source](https://github.c
 - customizable design for varying user expertise;
 - multi-GPU data parallelism support.
 
+## Installation
+Clone and build this repository from source
+
+```bash
+git clone https://github.com/Project-MONAI/MONAI.git
+python setup.py install
```

Comment:
forgot to check this just now, i think we need `cd MONAI/` before python setup.py, or `pip install -e MONAI/` works fine as well

Reply:
oh yes you are right